cmd/netobserv-ebpf-agent.go

```diff
@@ -39,22 +39,46 @@ func main() {
 
 	logrus.WithField("configuration", fmt.Sprintf("%#v", config)).Debugf("configuration loaded")
 
-	flowsAgent, err := agent.FlowsAgent(&config)
-	if err != nil {
-		logrus.WithError(err).Fatal("can't instantiate NetObserv eBPF Agent")
-	}
+	if config.EnablePano {
+		packetsAgent, err := agent.PacketsAgent(&config)
+		if err != nil {
```

Comment:
If this code is common can we move it to a single function so less duplication is present.